Cargo.toml

```diff
@@ -4,5 +4,6 @@ members = [
   "springql",
   "springql-core",
   "foreign-service",
+  "springql-deconfig",
```

Comment:
I like springql-config-parser better.

Someone may feel it's related to "cancellation of a configuration".
see: https://en.wiktionary.org/wiki/deconfigure